RELEASE.md

```diff
@@ -1,3 +1,13 @@
+# v2.0.0
+
+* Brand new UI & UX
+* Re-writes underlying App Architecture
```

Comment:
Is 'App Architecture' a particular directory or set of files? Otherwise, might be okay to not refer to it as a proper noun.